main.py

```diff
@@ -122,13 +121,17 @@ def parse_arguments():
     model = parser.add_argument_group('Learning options')
     models = ['Burgess']
     model.add_argument('-m', '--model',
-                       default=experiment_config['model'], choices=models,
+                       default=default_config['model'], choices=models,
                        help='Type of encoder and decoder to use.')
     model.add_argument('-z', '--latent-dim',
-                       default=experiment_config['latent_dim'], type=int,
+                       default=default_config['latent_dim'], type=int,
                        help='Dimension of the latent variable.')
 
     args = parser.parse_args()
+    
+    print(args)
```

Comment:
use logging instead of printing (easier to deactivate). The logger is instantiated in the main. SO in `main`: `logger.info(args)`

Reply:
Yeah also pretty sloppy. Nice spot.